Develop

vanjamilicevic opened the pull request:
Completed infinite scroll app. 

There is an issue with Unsplash API where limit of 50 requests is exceeded if used within an hour - which is small even for testing purposes.

I also wanted to put access key into environment variable, but was afraid to test it due to the small amount of requests that Unsplash provides.

vanjamilicevic committed:
Completed html, css and JS for infinite scroll app

vanjamilicevic committed:
Added Vercel links to README

vanjamilicevic committed:
Slightly changed API calls to avoid Unsplash rate limit

vanjamilicevic committed:
Changed number of images per load

elvira-quantox commented:
@vanjamilicevic Lepo izgleda, ako imaš vremena, ispravi sledeće:
- "Make sure that each image has a default alt text if it's not available from the response."
- lightbox funkcionalnost ("Upon clicking each image, show a higher resolution image in a lightbox").
- Probaj da izbegneš pojavu horizontalnog scroll-a

vanjamilicevic committed:
Changed alt, lightbox and disabled horizontal scroll

- Added img atribute that each image has default "alt" text if it is not
  available from the response
- Modified lightbox that showing images are in higher resolution
- Modified CSS in order to avoid horizontal scroll

vanjamilicevic committed:
Removing console logs

vanjamilicevic commented:
> @vanjamilicevic Lepo izgleda, ako imaš vremena, ispravi sledeće:
> 
> * "Make sure that each image has a default alt text if it's not available from the response."
> * lightbox funkcionalnost ("Upon clicking each image, show a higher resolution image in a lightbox").
> * Probaj da izbegneš pojavu horizontalnog scroll-a

Ispravljeno :)
